### PR TITLE
expand installer template to set extra SSH pub keys

### PR DESCRIPTION
**Pull request by manurodriguez**
##### SUMMARY
This will allow to specify multiple SSH key paths
to be applied at day1 and SSH can be performed
from multiple sources.

##### ISSUE TYPE

- Enhance role

##### Tests

- [x] TestBos2 - <https://www.distributed-ci.io/jobs/5670f29e-9995-4ebd-b69e-5e939b434b81/jobStates?sort=date
- [x] TestDallas - https://www.distributed-ci.io/jobs/b862acbb-c056-47b9-ae75-8b89498c2896/jobStates?sort=date

Test-Hints: no-check

### CODE REVIEW

File: roles/installer/templates/install-config.j2 [hunk 1]
```diff
@@ -185,7 +185,13 @@ platform:
 {% endfor %}
 {% endif %}
 pullSecret: '{{ pullsecret }}'
-sshKey: '{{ key }}'
+sshKey: |
```
Comment (tonyskapunk) [inline roles/installer/templates/install-config.j2:188]: Let's avoid an extra `\n` at the end
```suggestion
sshKey: |-
```

I'm still not sure where the other `\n`  in the config is coming from, see https://www.distributed-ci.io/jobs/bb8bee10-a34d-4883-b71d-e2b5008d62a9/files under `openshift_install_state.json` or `cluster-install-config.yml` and you'll see an extra `\n` or an empty space in between the `sshKey` and the extra keys.

Comment (manurodriguez) [reply roles/installer/templates/install-config.j2:188]: Fixed and thanks,  I believe the extra `\n` comes from a `b64decode` task here: https://github.com/redhatci/ansible-collection-redhatci-ocp/blob/main/roles/installer/tasks/25_create-install-config.yml#L17

I copied those tasks and test it in a quick play:
{code}
[dciteam@jumphost manny]$ ansible-playbook simple-play.yml 
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [Play to validate stuff] ****************************************************************************************************************************************************************************************************************

TASK [Verify that SSH key for dciteam exists] ************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [Get the contents of the ssh key for dciteam] *******************************************************************************************************************************************************************************************
ok: [localhost]

TASK [Set Fact for the ssh key of dciteam] ***************************************************************************************************************************************************************************************************
ok: [localhost]

TASK [Print key] *****************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDjHheL4PFaRRrKhTIoKyCgYs7MsZculexcWH06GQslSqH7y+dxgytXQdbn6FPpNWhjT53qe7d3nomJg9QkaV1BIdz0GRdYs7XC9no5urzHoHcH1UV/MeQ+4K7QbYGlzapRPKeSNlADGpD7VuFp9Kgx6oVUC/CeIZlVtOAd+kcOogWc1uvZxS/cE8vLS4SnSikhi51iLaXwjKDSY9eyzUO9mgqbL5Xl19V2LJCfB9nUsiHJphh3ss2JQBTj9E7Re7IsZAg1OCyIvHwr/yTJvC4jGB7wD7XtbwZ0CB9yNqkNGfwiUfPuDBvCfn/AgcOTRW7cy9uRrLc6ZMphWMaHbEpYGRwQAn4bU94BR51A3l8LBo/30RvzP9TdLHId10zLb4N2+4C1ran0yxTy3ZpLLrQ8kR0UhOi7+E5lBM4yibXsaMYAAaoVPEP4LMVPQ7VE/EnqxY+bzOBWltlPKDtZ8WxYNPo8CtAzJgJULGitg0V/v4uYwhqV9qRVXYhGqvVXSbk= dci-openshift-agent@jumphost.dfwt5g.lab\n"
}

PLAY RECAP ***********************************************************************************************************************************************************************************************************************************
localhost                  : ok=4    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0 
{code}

If I use the `trim` filter it's gone: `key: "{{ sshkey['content'] | b64decode | trim }}"` 

```
TASK [Print key] *****************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDjHheL4PFaRRrKhTIoKyCgYs7MsZculexcWH06GQslSqH7y+dxgytXQdbn6FPpNWhjT53qe7d3nomJg9QkaV1BIdz0GRdYs7XC9no5urzHoHcH1UV/MeQ+4K7QbYGlzapRPKeSNlADGpD7VuFp9Kgx6oVUC/CeIZlVtOAd+kcOogWc1uvZxS/cE8vLS4SnSikhi51iLaXwjKDSY9eyzUO9mgqbL5Xl19V2LJCfB9nUsiHJphh3ss2JQBTj9E7Re7IsZAg1OCyIvHwr/yTJvC4jGB7wD7XtbwZ0CB9yNqkNGfwiUfPuDBvCfn/AgcOTRW7cy9uRrLc6ZMphWMaHbEpYGRwQAn4bU94BR51A3l8LBo/30RvzP9TdLHId10zLb4N2+4C1ran0yxTy3ZpLLrQ8kR0UhOi7+E5lBM4yibXsaMYAAaoVPEP4LMVPQ7VE/EnqxY+bzOBWltlPKDtZ8WxYNPo8CtAzJgJULGitg0V/v4uYwhqV9qRVXYhGqvVXSbk= dci-openshift-agent@jumphost.dfwt5g.lab"
}

```

**Comment by manurodriguez** [comment]
recheck

File: roles/installer/templates/install-config.j2 (hunk 1 above)
Comment (tonyskapunk) [reply roles/installer/templates/install-config.j2]: Ah! There it was, I'm glad you found it!

**Review by tonyskapunk — APPROVED** [review]
LGTM!

**Pull request merged**